controllers/cluster_controller.go

```diff
@@ -422,7 +422,7 @@ func (c clusterDescendants) filterOwnedDescendants(cluster *clusterv1.Cluster) (
 		obj := o.(client.Object)
 		acc, err := meta.Accessor(obj)
 		if err != nil {
-			return nil
+			return nil // nolint:nilerr // We don't want to exit the EachListItem loop, just continue
```

Comment:
This looks like the kind of error which should never occur. Not sure what we should do in case it does occur.

Reply:
I guess this is a leftovers of v1alpha3 --> v1alpha4 changes.
You can now entirely drop the accessor and pass obj directly to util.IsOwnedByObject